jscomp/others/js_json.ml

```diff
@@ -52,6 +52,31 @@ let reifyType (type a) (x : 'a) : (a kind * a ) =
 
 let reify_type = reifyType 
 
+type tagged_t = 
```

Comment:
I'm curious about this naming convention. For one it's snake_case in a namespace which by typically uses camelCase, and for another I don't think I've seen the `_t` or any similar convention used for types anywhere else in the `Js` namespace.